src/battle_script_commands.c

```diff
@@ -4673,6 +4719,9 @@ static void Cmd_playanimation(void)
         return;
     }
     #endif
+    
+    //if (animId == B_ANIM_STATS_CHANGE && GetBattlerAbility(gActiveBattler) == ABILITY_CONTRARY)
+        //ReverseStatAnimId(argumentPtr);
```

Comment:
Is there a reason to keep this commented out `if`? :eyes:

Reply:
oops, no. That was an iteration on my first attempt that wasn't working. I'll remove it